server.js

```diff
@@ -0,0 +1,13 @@
+var express = require('express');
+var app = express();
+var path = require('path');
+
+app.set('port', (process.env.PORT || 5000));
+
+app.get('/', function (req, res) {
+    res.sendFile('index.html', { root: path.join(__dirname, '../vyapi/dist') });
```

Comment:
IMHO only the content of `dist` folder should go to heroku git not the entire repo. In that case you will not have this folder